README.md

```diff
@@ -73,6 +73,7 @@ Discussion should be open for ~1 week to give members time to review and contrib
  | EF Protocol Support | [Sam Wilson](https://github.com/SamWilsn/) | 1 |
  | EF Protocol Support | [Tim Beiko](https://github.com/timbeiko/) | 1 |
  | EF Protocol Support | [Trenton Van Epps](https://github.com/tvanepps/) | 1 |
+ | EF Research | [Arantxa Zapico](https://sites.google.com/view/arantxazapico/research) | 1 |
```

Comment:
```suggestion
 | EF Cryptography | [Arantxa Zapico](https://sites.google.com/view/arantxazapico/research) | 1 |
```
Should we list the official team? Or is it actually EF Research?

Reply:
I think we should do that for everyone else in EF Cryptography as well, and also for ARG and Consensus R&D. EF research is just an umbrella team, it doesn't make much sense for RIG to be listed separately but Cryptography within it